packages/web/src/generic.ts

```diff
@@ -106,7 +107,7 @@ function track(
   }
 
   if (!properties) {
-    window.va?.('event', { name });
+    window.va?.('event', { name, options });
```

Comment:
**suggestion(blocking):** While the user-facing property is `flags`, the analytics script expect [`flagsKeys`](https://github.com/vercel/front/pull/31293/files#diff-fe13bc40f2a9c7c1cbb5ae022240a9ae45689228be8f30f8eff3fd13bac828b6R19)